iphone/Maps/Categories/UIStackView+ArrangedSubviewWithSeparator.swift

```diff
@@ -40,64 +44,20 @@ extension UIStackView {
 }
 
 extension UIStackView.Separator {
-  static let `default` = UIStackView.Separator(thickness: CGFloat(1.0),
-                                               color: StyleManager.shared.theme?.colors.blackDividers,
-                                               insets: .zero,
-                                               position: .before)
   
-  func withInsets(_ insets: UIEdgeInsets) -> Self {
-    .init(thickness: thickness, color: color, insets: insets, position: position)
-  }
-}
-
-// MARK: - Add line
-private extension UIView {
-  enum LinePosition {
-    case leading, trailing, top, bottom
+  /// A default value returns separator configuration with a thickness of 1.0, zero insets, position "before" and color fetched from current system style.
+  static var `default`: Self {
+    .init(thickness: CGFloat(1.0),
+          color: StyleManager.shared.theme?.colors.blackDividers,
+          insets: .zero,
+          position: .before)
   }
   
-  func addLine(thickness: CGFloat,
```

Comment:
Looks like this commit can be squashed with the first one, right?

Do you have experience with git rebase/fixup/squashing and partial commit editing to clean up commit history?

Btw, [Fork](https://git-fork.com/) app is the best Git client for Mac.

Reply:
Sorry for that, i'm using fork and cleaned up commit history from probably 12 to 5 but miss that... will fix that!